src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -4051,21 +4051,23 @@
      *
      * <p>For example:</p>
      * <pre>
-     * public record myRecord () {}
+     * public record myRecord () {
```

Comment:
Make record follow naming convention for classes (upper camel case)

Reply:
done